cli-plugins/apps/getting-started/installation.hbs.md

```diff
@@ -0,0 +1,50 @@
+# Installation
+
+## Prerequisites
+
+Ensure that you installed or updated the core Tanzu CLI. For more information, see
```

Comment:
This should link to the installation instructions on vmware docs (now that they've been updated)

Here's the page that we want to link to, including the inline-anchor within the page (although we don't want to link directly to the TAP 1.6 version - we want this to be a durable link that's not tied to a specific TAP release) --- https://docs-staging.vmware.com/en/draft/VMware-Tanzu-Application-Platform/1.6/tap/install-tanzu-cli.html#cli-and-plugin

I'm not exactly how to format the URL though because the directory structure appears to be flattened somewhat when these docs get published to vmware-docs...

Please ask the docs writers to help you correctly set this URL up in the markup (@kcoriordan - can you help?)

Here's what I'd like this to be (but please update the link so we know it will work on the public docs site):

Ensure that you installed or updated the core Tanzu CLI. For more information, see [Install Tanzu CLI](https://docs-staging.vmware.com/en/draft/VMware-Tanzu-Application-Platform/1.6/tap/install-tanzu-cli.html#cli-and-plugin).

Reply:
changed to point to the latest tanzu cli installation instructions